src/snowflake/snowpark/functions.py

```diff
@@ -3399,6 +3401,7 @@ def sproc(
     session: Optional["snowflake.snowpark.Session"] = None,
     parallel: int = 4,
     statement_params: Optional[Dict[str, str]] = None,
+    execute_as: te.Literal["caller", "owner"] = "owner",
```

Comment:
hmm what's the different between Literal from typing and typing_extensions?

Reply:
Nothing, I thought it was introduced after 3.8. [I was wrong tho](https://docs.python.org/3/library/typing.html#typing.Literal).
> New in version 3.8.

I'll update my code.